README.md

```diff
@@ -4,6 +4,13 @@ The objective of this repository is hold the most common configuration for `verd
 
 > Feel free to contribute whether you considere any use case is relevant for the public in general.
 
+## Login 
+
+If you want to login into the Verdaccio instance created via these Docker Examples, please try:
+
+Username: jpicado
+Password: jpicado
+
```

Comment:
Sorry, I didn't show that this is the main README. This user-pass pair doesn't apply on all examples, As @juanpicado said in Discord, the README you should update is in _docker-local-storage-volume_ folder ([here](https://github.com/verdaccio/docker-examples/blob/master/docker-local-storage-volume/readme.md))

Also, there is other example that need this info, _https-portal-example_  ([here](https://github.com/verdaccio/docker-examples/blob/master/https-portal-example/README.md)). Please, can you add in this README also? Thanks :slightly_smiling_face: